management/user_test.go

```diff
@@ -80,13 +101,36 @@ func TestUser(t *testing.T) {
 		t.Logf("%v\n", uu)
 	})
 
-	t.Run("Delete", func(t *testing.T) {
-		err = m.User.Delete(auth0.StringValue(u.ID))
+	t.Run("GetRoles", func(t *testing.T) {
+		var roles []*Role
+		roles, err = m.User.GetRoles(auth0.StringValue(u.ID))
+		if err != nil {
+			t.Error(err)
+		}
+		t.Logf("%v\n", roles)
+	})
+
+	t.Run("AssignRoles", func(t *testing.T) {
+		roles := []*Role{r1, r2}
+		err = m.User.AssignRoles(auth0.StringValue(u.ID), roles...)
```

Comment:
This can also be written using variadic args:

```
m.User.AssignRoles(auth0.StringValue(u.ID), r1, r2)
```

Reply:
Nice. Thanks!